screens/Viewer.tsx

```diff
@@ -40,35 +42,53 @@ export function Viewer(props: Props) {
   const [page, setPage] = useState(0);
   const [pageRate, setPageRate] = useState(0);
 
+  const safeAreaInsets = useSafeAreaInsets();
+
   useEffect(() => {
     setPageRate(props.initialPageRate || 0);
   }, [props.initialPageRate]);
 
+  const navigation = useNavigation();
+
   if (episode === undefined) {
     return <Text>loading...</Text>;
   }
 
   return (
     <View style={{ ...styles.container, backgroundColor: colors.background }}>
-      <ReaderBrowser
-        body={episode.body}
-        page={page}
-        pageMax={pageMax}
-        pageRate={pageRate}
-        onTap={() => {
-          setIsShowMenu(!isShowMenu);
+      <ViewerNavigation
+        title={episode.title}
+        onPress={() => {
+          navigation.goBack();
         }}
-        onPullPrev={() => {}}
-        onPullNext={() => {}}
-        onUpdatePageMax={(pageMax) => {
-          setPageMax(pageMax);
-        }}
-        onChangePage={(page) => {
-          setPage(page);
-          // TODO: updateBookmark
-          // https://github.com/rutan/compac-reader/blob/9d90e67949358ba3ecba6bbc43ab5933a94e5b83/src/view/screen/reader/index.js#L110-L117
+        style={{
+          ...styles.header,
+          paddingTop: safeAreaInsets.top,
+          // height: safeAreaInsets.top + 44,
+          opacity: isShowMenu ? 1 : 0,
         }}
       />
+      <SafeAreaView style={styles.container}>
```

Comment:
ReaderBrowser' HTML is not yet compatible with Safe Area. ReaderBrowser should be out of the SafeAreaView because an HTML's background color does not affect Safe Area. (The view has white background on top and bottom)